webrtc.html

```diff
@@ -9849,6 +9851,40 @@ <h2>Methods</h2>
         </section>
       </div>
     </section>
+    <section>
+      <h3>canInsertDTMF update</h3>
+      <p>To <dfn>update the [[\CanInsertDTMF]]</dfn> slot of an <code><a>RTCDTMFSender</a></code>
+      instance <var>dtmfSender</var>, the user agent MUST queue a task that runs the following steps:</p>
+      <ol>
+        <li>Let <var>sender</var> be the <code><a>RTCRtpSender</a></code> associated with
+        <var>dtmfSender</var>.</li>
+        <li>Let <var>transceiver</var> be the <code><a>RTCRtpTransceiver</a></code>
+        associated with <var>sender</var>.</li>
+        <li>Let <var>pc</var> be the <code><a>RTCPeerConnection</a></code> associated with
+        <var>transceiver</var>.</li>
+        <li>Check whether <var>pc</var>'s <a>[[\ConnectionState]]</a> slot does not have the value
+        <code>"connected"</code> or <code>"completed"</code>. If so, set <var>dtmfSender</var>'s
```

Comment:
I can't find `"completed"` in the connectionState [enum](https://w3c.github.io/webrtc-pc/#dfn-connection-state)

Maybe it's only me, but I think it would clearer if put something like
`<li>Check whether <var>pc</var>'s <a>[[\ConnectionState]]</a> slot has the value
        <code>"connected". If not, set <var>dtmfSender</var>'s